scripts/lock.sh

```diff
@@ -2,12 +2,10 @@
 # file: lock.sh
 # description: Lock dependencies and export requirements.
 
-echo "# THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements.txt
-echo "# THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements-dev.txt
+echo -e "# THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements.txt
```

Comment:
Pardon my ignorance, but what does this do? And why is it needed?

I'm happy to merge it if you think it'll improve the script.